forms/models/form.js

```diff
@@ -17,9 +17,17 @@ define(function (require) {
   var Form;
   var invalidWrapperFn, isSubForm;
 
+  function isVisible(elementModel){
+    if ( !elementModel.has('hidden')){
```

Comment:
I doubt it, I have a feeling that it was for stopping elements that dont have the hidden field showing a console error about 'undefined', but I'll remove it. see how it goes. On the surface it doesn't seem necessary at all.